.github/workflows/test.yml

```diff
@@ -216,9 +216,13 @@ jobs:
             .go
             .mod
             .sum
-      - name: build image
-        run: |
-          make build-docker-local-simapp
+      - name: build and push image
+        uses: docker/build-push-action@v1.1.0
+        with:
+          repository: cosmossdk/simd-env
+          username: tenderbot
+          password: ${{ secrets.TENDERBOT_TOKEN }}
+          dockerfile: contrib/images/simd-env/Dockerfile
```

Comment:
we should sync on where images are being pushed. nightly builds get pushed to interchainio/simapp, maybe we push there as well for this but name the repo simapp-dev